src/app/stack/card-details/component-information/component-snippet/component-snippet.component.html

```diff
@@ -120,20 +120,31 @@ <h4>Details of the dependency: <span class="component-name" [innerText]="compone
                     <span *ngFor="let cve of component?.public_vulnerabilities" class="wrapper">
                         <!-- <span class="severity"
                             [ngClass]="{'critical':cve.severity == 'critical', 'high':cve.severity == 'high','medium':cve.severity == 'medium','low':cve.severity == 'low'}"> -->
-                        <span *ngIf="cve.severity == 'critical'" class="circle critical" [innerText]="'C'" title="Critical"></span>
+                        <span *ngIf="cve.severity == 'critical'" class="circle critical" [innerText]="'C'"
+                            title="Critical"></span>
                         <span *ngIf="cve.severity == 'high'" class="circle high" [innerText]="'H'" title="High"></span>
-                        <span *ngIf="cve.severity == 'medium'" class="circle medium" [innerText]="'M'" title="Medium"></span>
+                        <span *ngIf="cve.severity == 'medium'" class="circle medium" [innerText]="'M'"
+                            title="Medium"></span>
                         <span *ngIf="cve.severity == 'low'" class="circle low" [innerText]="'L'" title="Low"></span>
-                        <!-- <span [innerText]="cve.severity"></span> -->
-                        <!-- </span> -->
-                        <a [href]="getUrl(cve.url, tabType)" target="_blank">
-                            <span class="name" [innerText]="cve.title" [tooltip]="cve.id" title="command/ctrl + click"></span>
-                        </a>
+                        <span class="custom-tooltip name"><a [href]="getUrl(cve.url, tabType)" target="_blank">{{cve.title}}</a>
+                            <div class="custom-tooltip-text tooltip-content">
+                                <div class="row">
+                                    <div class="col-sm-5">CVSS Score:</div>
+                                    <div class="col-sm-7">{{cve.cvss}}</div>
+                                </div>
+                                <div class="row">
+                                    <div class="col-sm-4">Snyk ID:</div>
+                                    <div class="col-sm-8 no-wrap"><a [href]="getUrl(cve.url, tabType)" target="_blank" [innerText]="cve.id"></a></div>
+                                </div>
+                                <div class="row">
+                                    <div class="col-sm-12 engrave">(cmd/ctrl + click)</div>
+                                </div>
+                            </div>
+                        </span>
```

Comment:
Can we define the custom tool tip as a separate Angular component? It would help us to reuse it somewhere else too.(e.g from Dependency card?)

Reply:
Ya, can do it, Let me try and give an update on this

Reply:
@arajkumar I tried generating a new component but it's giving me the below error:
"The generate command requires to be run in an Angular project, but a project definition could not be found."

Reply:
I tried these solutions but it gives me the same issue : 
https://stackoverflow.com/questions/53096996/angular-cli-error-the-serve-command-requires-to-be-run-in-an-angular-project-b